src/app_charts/base/app_management_test.sh

```diff
@@ -14,7 +14,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-HELM="${TEST_SRCDIR}/_main/external/_main~non_module_deps~kubernetes_helm/helm template"
+if [[ -x "${TEST_SRCDIR}/_main/external/_main~non_module_deps~kubernetes_helm/helm" ]] ; then
+  HELM="${TEST_SRCDIR}/_main/external/_main~non_module_deps~kubernetes_helm/helm template"
+elif [[ -x "${TEST_SRCDIR}/+non_module_deps+kubernetes_helm/helm" ]] ; then
+  HELM="${TEST_SRCDIR}/+non_module_deps+kubernetes_helm/helm template"
+else
```

Comment:
can we do the same in deploy.sh? Any idea why my find approach failed? Btw. my find approach worked locally, but failed when I manually triggered the integration tests action workflow.

Reply:
Maybe less copy'n'paste if we write:
```
HELM="${TEST_SRCDIR}/_main/external/_main~non_module_deps~kubernetes_helm/helm"
if [[ ! -x "${HELM}" ]]; then
  HELM="${TEST_SRCDIR}/+non_module_deps+kubernetes_helm/helm template"
fi
if [[ ! -x "${HELM}" ]]; then
  echo >&2 "Failed to locate helm in ${TEST_SRCDIR}."
  exit 1
fi
```

Reply:
Done. I copied it to deploy.sh, but I was too lazy to reimplement the logic in Go, so I just left a comment that hopefully someone will find when they update the Bazel pin.